py/client-ticking/setup.py

```diff
@@ -5,89 +5,99 @@
 import pathlib
 import platform
 
+from Cython.Build import cythonize
+
 # Note: pkg_resources is deprecated https://setuptools.pypa.io/en/latest/pkg_resources.html, and it is suggested
 # to use an external library `packaging`. From the context of building a wheel though, we'd prefer to not have to
 # install extra dependencies, at least until we can more properly manage the build environment (pyproject.toml).
 # TODO(deephaven-core#2233): upgrade setup.py to pyproject.toml
-from pkg_resources import parse_version
-from setuptools import find_packages, setup, Extension
-from Cython.Build import cythonize
+from packaging.version import parse as parse_version
```

Comment:
We've changed from pkg_resources to packaging, which seems to mean this TODO has been completed and is now obsolete.